contrib/scripts/bootc-destructive-cleanup

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
WDYT if we called this e.g. `fedora-bootc-destructive-cleanup`, leaving space for someone to put in an arch/deb/etc version later?